cranelift/codegen/src/machinst/vcode.rs

```diff
@@ -563,22 +563,20 @@ impl<I: VCodeInst> VCode<I> {
             let (start, end) = self.block_ranges[block as usize];
             for iix in start..end {
                 let srcloc = self.srclocs[iix as usize];
-                if srcloc != cur_srcloc {
-                    if !cur_srcloc.is_default() {
+                if Some(srcloc) != cur_srcloc {
```

Comment:
Perhaps swap the arguments of the `!=` around, to make this a bit easier to read?